src/layout/section/cardsSection/index.tsx

```diff
@@ -1,52 +1,56 @@
 import styled from 'styled-components'
 import { color, space, responsiveBreakpoints, componentSizes } from '_utils/branding'
 
-import TripsSection from './TripsSection'
+import CardsSection from './CardsSection'
 
 const wrapperHorizontalPadding = space.xl
 
-const StyledTripsSection = styled(TripsSection)`
+const StyledCardsSection = styled(CardsSection)`
   & {
     background-color: ${color.lightBackground};
   }
 
-  & .kirk-tripsSection {
+  & .kirk-cardsSection {
     margin: auto;
     display: flex;
     padding: ${space.l} ${wrapperHorizontalPadding};
     box-sizing: border-box;
-    overflow: auto; /* Make TripCards scrollable horizontally */
+    overflow: auto; /* Make Cards scrollable horizontally */
     -ms-overflow-style: none; /* Remove scrollbar visually IE 10+ */
     scrollbar-width: none; /* Remove scrollbar visually Firefox */
     scroll-snap-type: x mandatory;
     scroll-padding: ${space.xl};
   }
 
-  & .kirk-tripsSection .kirk-tripCard {
+  & .kirk-cardsSection .kirk-tripCard,
```

Comment:
Can we have a reused util class between trip and qr cards to avoid all these duplicated rules ?

Reply:
Welle technically they both use a `<Card  />` as root so I could use the `.kirk-card` class, I'm  gonna take a look  you're right